assets/src/BlockEditorValidation.js

```diff
@@ -10,37 +10,38 @@ let canPublish = true;
 
 export const blockEditorValidation = () => {
   subscribe(() => {
-    const blocks = select( 'core/block-editor' ).getBlocks();
+    const blocks = select('core/block-editor').getBlocks();
 
     const currentMessages = [];
-    const invalidBlocks = blocks.reduce( (invalidBlocks, block ) => {
+    // eslint-disable-next-line no-shadow
+    const invalidBlocks = blocks.reduce((invalidBlocks, block) => {
       // Normally `blocks` contains a valid list of blocks, however it can happen that one of them is `null` in rare
       // cases. It happened to me once while running with WordPress 5.8.1 and undoing multiple edits. This made the
       // editor crash while it's trying to access `block.name`.
       if (!block) {
-        return;
+        return null;
```

Comment:
Are you sure this doesn't cause a crash? 🤔 According to the comment above `it can happen that one of them is null` was the issue prevented here